requirements.test.txt

```diff
@@ -1,4 +1,4 @@
-.
+-e .
```

Comment:
What's the significance of making the packages in the install_requires editable?